contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtConfigGroup.java

```diff
@@ -50,9 +51,40 @@
 
 	@SuppressWarnings("deprecation")
 	public static DrtConfigGroup get(Config config) {
-		return (DrtConfigGroup)config.getModule(GROUP_NAME);
+//		return (DrtConfigGroup)config.getModule(GROUP_NAME);
+		return ConfigUtils.addOrGetModule( config, DrtConfigGroup.class ) ;
+		// yyyy I think that this method should be inlined and then removed to become consistent with how it is done elsewhere.  kai, aug'19
+		// yy I have to say that I am not sure if the method should exist at all.  It seems to me that DrtConfigGroup can exist both at the base level
+		// _and_ as one of many such config groups inside MultiModeDrtConfigGroups.  Our experience with such dual use approaches is not so good.  kai,
+		// aug'19
 	}
```

Comment:
This method cannot have a side effect (i.e. creating a module if missing), otherwise checks in `MultiModeDrtConfigGroup` will get broken.

Re disallowing a standalone `DrtConfigGroup`: probably the only risk is that most people use a single DRT setup. So by doing this, we will break lots of scenarios (which we do from time to time anyway) and we will make the typical setup a bit more complicated (no injection of `DrtConfigGroup`, and obtaining it by providing additional argument: `String drtServiceName`). But maybe worth doing. Will add this to my task list for the upcoming (remote) DevMeet:-)

Reply:
Mhm, I'm not against it, but maybe we should then get the naming right:
MultiModeDrtConfigGroup --> let's name it drt or DrtConfigGroup or whatever.
DrtConfigGroup --> rename it to DrtOperatorConfigGroup, cause that's what it is.